assimilation_code/modules/assimilation/assim_tools_mod.f90

```diff
@@ -976,7 +952,8 @@ subroutine obs_increment(ens_in, ens_size, obs, obs_var, obs_kind, obs_inc, &
 !--------------------------begin algorithm_info control block-----------------
 ! More flexible abilities to control the observation space increments are 
 ! available with this code block. It gets information about the increment method
-! for the current observation is use_algorithm_info_mod is set to true in the namelist.
+! for the current observation.
+
```

Comment:
The comments  from 

```
!--------------------------begin algorithm_info control block-----------------

!----------------------------end algorithm_info control block-----------------
```

are any of these needed with the code in this pull request?